fauxfactory/__init__.py

```diff
@@ -57,7 +57,7 @@ def _make_unicode(data):
         cast is necessary, a copy of ``data`` is returned.
 
     """
-    if sys.version_info.major == 2:
+    if sys.version_info[0] == 2:
```

Comment:
I do not have a Python 2.6 system available for manual testing. But the documentation indicates that major, minor, micro, releaselevel, and serial are available. See [here](https://docs.python.org/2.6/library/sys.html#sys.version_info).